api/lanes.py

```diff
@@ -339,44 +339,27 @@ def lane_for_small_collection(_db, languages):
 def lane_for_other_languages(_db, exclude_languages):
     """Make a lane for all books not in one of the given languages."""
 
-    YA = Classifier.AUDIENCE_YOUNG_ADULT
-    CHILDREN = Classifier.AUDIENCE_CHILDREN
+    language_lanes = []
+    other_languages = Configuration.tiny_collection_languages()
 
-    common_args = dict(
-        exclude_languages=exclude_languages,
-        genres=None,
-    )
-
-    adult_fiction = Lane(
-        _db, 
-        full_name="Adult Fiction",
-        display_name="Fiction",
-        fiction=True, 
-        audiences=Classifier.AUDIENCES_ADULT,
-        **common_args
-    )
-    adult_nonfiction = Lane(
-        _db, full_name="Adult Nonfiction", 
-        display_name="Nonfiction",
-        fiction=False, 
-        audiences=Classifier.AUDIENCES_ADULT,
-        **common_args
-    )
-
-    ya_children = Lane(
-        _db, 
-        full_name="Children & Young Adult", 
-        fiction=Lane.BOTH_FICTION_AND_NONFICTION,
-        audiences=[YA, CHILDREN],
-        **common_args
-    )
+    for language_set in other_languages:
```

Comment:
It should be the big scrolling list of all books not in exclude_languages. But I just checked, and it turns out the lane url is wrong again when this happens, so I'll look into that.